backend/src/test/java/com/pickpick/workspace/application/WorkspaceServiceTest.java

```diff
@@ -0,0 +1,99 @@
+package com.pickpick.workspace.application;
+
+import static com.pickpick.fixture.MemberFixture.KKOJAE;
+import static com.pickpick.fixture.WorkspaceFixture.JUPJUP;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.pickpick.channel.domain.Channel;
+import com.pickpick.channel.domain.ChannelRepository;
+import com.pickpick.exception.workspace.WorkspaceDuplicateException;
+import com.pickpick.member.domain.Member;
+import com.pickpick.member.domain.MemberRepository;
+import com.pickpick.support.DatabaseCleaner;
+import com.pickpick.workspace.domain.Workspace;
+import com.pickpick.workspace.domain.WorkspaceRepository;
+import java.util.List;
+import java.util.Optional;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class WorkspaceServiceTest {
+
+    @Autowired
+    private WorkspaceService workspaceService;
+
+    @Autowired
+    private MemberRepository members;
+
+    @Autowired
+    private WorkspaceRepository workspaces;
+
+    @Autowired
+    private ChannelRepository channels;
+
+    @Autowired
+    private DatabaseCleaner databaseCleaner;
+
+    @AfterEach
+    void tearDown() {
+        databaseCleaner.clear();
+    }
+
+    @DisplayName("워크스페이스 초기화 시 워크스페이스를 저장한다.")
+    @Test
+    void registerWorkspace() {
+        // given
+        Workspace jupjup = JUPJUP.create();
+
+        Optional<Workspace> beforeSave = workspaces.findBySlackId(jupjup.getSlackId());
+
+        // when
+        workspaceService.registerWorkspace(KKOJAE.getCode());
+
+        Optional<Workspace> afterSave = workspaces.findBySlackId(jupjup.getSlackId());
+
+        // then
+        assertThat(beforeSave).isEmpty();
+        assertThat(afterSave).isNotEmpty();
```

Comment:
```suggestion
        assertThat(afterSave).isPresent();
``` 
오 suggestion 기능 처음써보는데 좋네요